packages/embed-chart/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "name": "@deephaven/embed-chart",
+  "version": "0.15.4",
+  "description": "Deephaven Embedded Chart",
+  "author": "Deephaven Data Labs LLC",
+  "license": "Apache-2.0",
+  "type": "module",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/deephaven/web-client-ui.git",
+    "directory": "packages/embed-chart"
+  },
+  "homepage": ".",
+  "main": "public/index.js",
+  "files": [
+    "build"
+  ],
+  "dependencies": {
+    "@babel/core": "^7.16.0",
+    "@deephaven/chart": "file:../chart",
+    "@deephaven/components": "file:../components",
+    "@deephaven/icons": "file:../icons",
+    "@deephaven/jsapi-shim": "file:../jsapi-shim",
+    "@deephaven/jsapi-utils": "file:../jsapi-utils",
+    "@deephaven/log": "file:../log",
+    "@fortawesome/fontawesome-svg-core": "1.2.36",
+    "@fortawesome/react-fontawesome": "^0.1.15",
+    "@types/jest": "^26.0.24",
+    "@types/node": "^16.11.7",
+    "@types/react": "^17.0.2",
+    "@types/react-dom": "^17.0.9",
+    "classnames": "^2.3.1",
+    "deep-equal": "^2.0.5",
+    "fira": "mozilla/fira#4.202",
+    "react": "^17.0.2",
+    "react-dom": "^17.0.2",
+    "react-scripts": "4.0.3",
+    "sass": "^1.39.0",
+    "shortid": "^2.2.16",
+    "typescript": "~4.5.4"
```

Comment:
Technically these are all dev dependencies. Since we bundle applications, nothing except the bundle is needed. If you were to install w/ `npm install --production`, things would be fine if none of these dependencies were downloaded and you only got the build/dist folder

At the very least, babel, types, react-scripts, sass, typescript are dev deps